scripts/README.md

```diff
@@ -15,6 +17,17 @@ Run the following script to provision the infrastructure and deploy the API and
                              -subscription <subscription-id>
 ```
 
+## Using ACA
+
+Deployment using ACA instead of AKS requires the addition of the argument `-deployAks 0` to the command line call.
```

Comment:
I would like to make ACA deployment the default and have the user enter some value to optionally deploy to AKS instead.

Reply:
This has been updated.